ra_aid/tools/memory.py

```
@@ -224,308 +255,357 @@ def emit_key_snippets(snippets: List[SnippetInfo]) -> str:
+
     Note:
         Entries can be retrieved with get_work_log() as markdown formatted text.
     """
     from datetime import datetime
-    entry = WorkLogEntry(
-        timestamp=datetime.now().isoformat(),
-        event=event
-    )
-    _global_memory['work_log'].append(entry)
+
+    entry = WorkLogEntry(timestamp=datetime.now().isoformat(), event=event)
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)